src/components/graph-view.js

```diff
@@ -891,38 +891,44 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
             selectedNodeObj: null,
           });
           onSelect && onSelect({ nodes: null, edges: null });
+        } else {
+          // Position of moved node is at mouse location,
+          // all other nodes are relative to the moved node.
+          selected?.nodes?.forEach((current, key) => {
+            const selectedNodeID = key;
 
-          return;
-        }
-
-        // Position of moved node is at mouse location,
-        // all other nodes are relative to the moved node.
-        selected?.nodes?.forEach((current, key) => {
-          const selectedNodeID = key;
-
-          // skip the moved node
-          if (selectedNodeID === node[nodeKey]) {
-            return;
-          }
+            // skip the moved node
+            if (selectedNodeID === node[nodeKey]) {
+              return;
+            }
 
-          const selectedNodeMapNode = this.getNodeById(selectedNodeID);
+            const selectedNodeMapNode = this.getNodeById(selectedNodeID);
 
-          if (!selectedNodeMapNode) {
-            // this should never happen
-            return;
-          }
+            if (!selectedNodeMapNode) {
+              // this should never happen
+              return;
+            }
 
-          const selectedNode = selectedNodeMapNode.node;
+            const selectedNode = selectedNodeMapNode.node;
 
-          const newX = position.x + ((selectedNode.x || 0) - originalX);
-          const newY = position.y + ((selectedNode.y || 0) - originalY);
+            const newX = position.x + ((selectedNode.x || 0) - originalX);
+            const newY = position.y + ((selectedNode.y || 0) - originalY);
 
-          selectedNode.x = newX;
-          selectedNode.y = newY;
+            selectedNode.x = newX;
+            selectedNode.y = newY;
 
-          this.asyncRenderNode(selectedNode);
-        });
+            this.asyncRenderNode(selectedNode);
+          });
+        }
       }
+
+      // force the state to update with the new node information
+      this.setState({
+        nodesMap: {
+          ...nodesMap,
+        },
+        componentUpToDate: false,
+      });
```

Comment:
Setting the state here is the right thing to do because we've modified the state by setting the x,y positions on the nodes.